deploy.cue

```diff
@@ -18,6 +18,7 @@ import (
 	}
 	"mantis-staging": jobs: {
 		miner: #miner
+		passive: #passive
```

Comment:
```suggestion
		passive: #passive  & {#count: 1}
```

Reply:
As far as I understand this was added to allow the RPC20 test to run properly. So I think @krzysztofpaliga-iohk may be able to say better if one node is enough. What do you think Chris?

Reply:
For the ERC20 tests to run 1 passive node is enough

Reply:
Thanks @krzysztofpaliga-iohk. I fixed the config @nrdxp